src/module.js

```diff
@@ -83,7 +83,15 @@
  * });
  * </pre>
  */
-angular.module('stormpath',['stormpath.CONFIG','stormpath.auth','stormpath.userService'])
+angular.module('stormpath',[
```

Comment:
Inconsistent. Missing space between `',` and `['`. Other module declarations have a space. Or if not having a space is the "consistent" way to go, then the other module declarations are the ones that are inconsistent.